dojo/test/views.py

```diff
@@ -633,19 +633,19 @@ def re_import_scan_results(request, tid):
     form = ReImportScanForm()
     jform = None
     jira_project = jira_helper.get_jira_project(test)
-    push_all_jira_issues = jira_helper.is_push_all_issues(test)
+    push_all_jira_issues = jira_project.push_all_issues if jira_project else False
```

Comment:
the idea was to have the logic around push_all_jira_issues in one place or at least in the helper, so I don't think this change is needed.

Reply:
It's redundant. `is_push_all_issue` just calls `get_jira_project` to check the db field `push_all_issues`. The object is already here so why not use it?

Reply:
if the internals of is_push_all_issues change, we would have to update all the places where it is used.

Reply:
I would hope the db fields don't change often enough for that to be an issue 😉 perhaps it can be used as a deterrent to leave them alone. But if you insist it's to be done this way then let's merge.

Reply:
well at some point it was considered to make the `push_all_issues` on product level always overrule that on the engagement level. another thing could be the api where there could be logic around the push_to_jira coming in from the api request needs to be combined with the push_all_issue flag in the db. Does the incoming request overrule that, or not? Logic like that I'd like to have it in only 1 place

Reply:
Alright fair enough.